src/geth.rs

```diff
@@ -0,0 +1,197 @@
+use crate::jsonrpc;
+use anyhow::Context;
+use asset::Erc20Quantity;
+use comit::{
+    asset::{self, ethereum::TryFromWei},
+    ethereum::{Address, Bytes, ChainId, Hash, Transaction, TransactionReceipt},
+};
+use ethereum_types::U256;
+use num::{BigUint, Num};
+use serde_hex::{CompactPfx, SerHex};
+
+#[derive(Debug, Clone)]
+pub struct Client {
+    rpc_client: jsonrpc::Client,
+    jsonrpc_version: String,
```

Comment:
This is a constant can it be declared at such? (scope can be this module or the impl block of Client).

Reply:
It's 1.0 for bitcoind and 2.0 for geth :(

Reply:
Yes, then declare a constnant in `geth.rs` and another one in `bitcoind.rs`